.github/workflows/open-develop-pr.yaml

```diff
@@ -0,0 +1,35 @@
+##
+## Auto-opens a PR from main -> develop after a release has been published.
+##
+
+name: Open Develop PR
+
+on:
+  release:
+    types:
+      - released
+  workflow_dispatch:
+
+jobs:
+  run:
+    name: Upsteam develop branch
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Clarinet
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Open pull request
+        uses: repo-sync/pull-request@v2
+        with:
+          source_branch: main
+          destination_branch: develop
+          pr_title: "chore: update develop branch"
+          pr_body: |
+            :robot: This is an automated pull request created from a new release in [clarinet](https://github.com/hirosystems/clarinet/releases).
```

Comment:
```suggestion
            :robot: This is an automated pull request created from a new release in [clarity-repl](https://github.com/hirosystems/clarity-repl/releases).
```

Reply:
Fixed in 1fe39ff438e7da97171d6a7e4685854debaddea5. Thanks Charlie!